hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/read/TestStreamBlockInputStream.java

```diff
@@ -280,4 +289,38 @@ private void testReadEmptyBlock() throws Exception {
       assertEquals(-1, keyInputStream.read());
     }
   }
+
+  @Test
+  public void testCustomStreamReadConfigIsApplied() throws Exception {
+    // Arrange: create a config with non-default values
+    OzoneConfiguration conf = new OzoneConfiguration();
+    conf.set("ozone.client.stream.read.pre-read-size", "67108864");
+    conf.set("ozone.client.stream.read.response-data-size", "2097152");
+    conf.set("ozone.client.stream.read.timeout", "5s");
+
+    OzoneClientConfig clientConfig = conf.getObject(OzoneClientConfig.class);
+
+    // Sanity check
+    assertEquals(Duration.ofSeconds(5), clientConfig.getStreamReadTimeout());
```

Comment:
To test whether `OzoneClientConfig` applies the new config keys correctly, add test case in existing `TestOzoneClientConfig`.